### PR TITLE
Add a README.md

### PR DESCRIPTION
**Pull request by joshmiller83**
:bowtie:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+---------------------------
+
+This is a non-functional dev version. You're welcome to interact with us in the
+issue queue and submit patches / pull requests. 
+
+[Issue Tracker](https://drupal.org/project/issues/commerce)
+
+Getting Started
+---------------
+
```
Comment (rszrama) [inline README.md:17]: The links and text in this section are more specific to Commerce 1.x than 2.x. I'm not sure if we should try to fix it or just leave it out for now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+tracker and find support in IRC (#drupal-commerce on irc.freenode.net) where we
+try to be very responsive.
+
+[Learn more](https://www.drupalcommerce.org) | 
+[Developer Guide](http://www.drupalcommerce.org/developer-guide) | 
+[Issue Tracker](http://drupal.org/project/issues/commerce)
+
+How to Install
+--------------
+
```
Comment (rszrama) [inline README.md:34]: This is also more specific to Commerce 1.x. Nothing in Commerce 2.x at the moment has a dependency on these modules, and some don't even have Drupal 8 versions. Additionally, modules are installed to the root /modules directory now on Drupal 8. I think I'd just say you install a standard installation of Drupal 8, put this project into /modules, and then enable the desired modules from /admin/modules. We can add dependencies as they actually come in.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   - Tree              https://www.drupal.org/project/tree
+   - Addressfield      https://www.drupal.org/project/addressfield
+   - Rules             https://www.drupal.org/project/rules
+   - Composer Manager  https://www.drupal.org/project/composer_manager
+   - Search API        https://www.drupal.org/project/search_api
+2. Go to the Module page at Administer > Modules /admin/modules and enable them.
+3. ...
+4. Profit!
+
+Included Libraries
```
Comment (rszrama) [inline README.md:45]: "Related Libraries" (as they're not actually included in this repository). Also, anywhere below where you've called them components, call them libraries.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+------------------
+
+For the 2.x branch of Drupal Commerce, Commerce Guys has moved some of the logic
+out of the Drupal world and into the greater PHP community. This effort has
+produced three unique eCommerce-related PHP libraries (please star on github):
+
+###[Address](https://github.com/commerceguys/address)
+
+This component stores and manipulates postal addresses, more precisely the kind of
```
Comment (rszrama) [inline README.md:54]: The library actually doesn't manage the storage of address data. I see this came from the readme of the address library itself, so I suppose we can leave it here for now.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Zone checks if an address matches a set of conditions.
+
+###[Pricing](https://github.com/commerceguys/pricing)
+
+A component for managing prices, taxes, discounts, fees.
+
+Maintainers
+-----------
+
+Maintained by Ryan Szrama and Bojan Zivanovic of Commerce Guys.
```
Comment (rszrama) [inline README.md:68]: Needs markdown links.

**Comment by joshmiller83** [comment]
Closing this pull request and will make some changes, and create a new one. :boar:

**Pull request closed without merging**